src/rules/top-level-interop.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @fileoverview Rule to ensure top-level items are either an array or ojbect.
+ * @author Joe Hildebrand
+ */
+
+export default {
+	meta: {
+		type: /** @type {const} */ ("problem"),
+
+		docs: {
+			description:
+				"Disallow JSON top-level items are not an array or object",
```

Comment:
We generally use "require" unless the rule begins with `no-`.

```suggestion
				"Require the top-level value to be an array or object",
```